api/validator_test.go

```diff
@@ -12,25 +12,81 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type bankNumberParameter struct {
+	input    models.BankNumber
+	expected interface{}
+}
+
+type bankNumberExpectedParameter struct {
+	code    int
+	message string
+}
+
+var validationBanksAcceptedRulesAndFees = []bankNumberParameter{
+	{input: models.Caixa, expected: true},
+	{input: models.Stone, expected: true},
+	{input: models.BancoDoBrasil, expected: false},
+	{input: models.Santander, expected: false},
+	{input: models.Citibank, expected: false},
+	{input: models.Itau, expected: false},
+	{input: models.JPMorgan, expected: false},
+	{input: models.Pefisa, expected: false},
```

Comment:
@davidsmedeiros Faltou o bradesco 
```suggestion
	{input: models.Pefisa, expected: false},
	{input: models.Bradesco, expected: false},

```

Reply:
done